### ISSUE TITLE
Error on running Get-MgDirectoryDeletedItem without a parameters. Runs successfully when a DirectoryObjectId is specified.

### ISSUE THREAD

**Issue by msewaweru** [microsoftgraph/msgraph-sdk-powershell#832]
Get-MgDirectoryDeletedItem : Searches against this resource are not supported. Only specific instances can be queried.
At line:1 char:1
+ Get-MgDirectoryDeletedItem
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: ({ Top = , Skip ...ndProperty =  }:<>f__AnonymousType25`8) [Get-MgDirectoryDeletedItem_List1], RestException`1
    + FullyQualifiedErrorId : Request_UnsupportedQuery,Microsoft.Graph.PowerShell.Cmdlets.GetMgDirectoryDeletedItem_List1 
[AB#10918](https://microsoftgraph.visualstudio.com/0985d294-5762-4bc2-a565-161ef349ca3e/_workitems/edit/10918)

**Comment by peombwa** [comment]
Thanks for surfacing this. This isn't annotated in the metadata to tell us if it is supported or not. We will remove in the next release.

**Comment by peombwa** [comment]
This is a valid command; we are the ones who do not support OData cast. The commands should be used as shown [here](https://docs.microsoft.com/en-us/graph/api/directory-deleteditems-list?view=graph-rest-1.0&tabs=http) to list recently deleted directory items by type.

We should consider adding a mandatory `-ObjectType` parameter to `list` parameterSet that customers can use to specify the type of objects to list. i.e., `Get-MgDirectoryDeletedItem -ObjectType microsoft.graph.user`.

Depends on #19.
Related to #423.

**Comment by peombwa** [comment]
Closing as duplicate of https://github.com/microsoftgraph/msgraph-sdk-powershell/issues/423.
